content/account/index.textile

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Whilst it does help focus on key words, we don't tend to use bold unless it's a UI interaction, or in a table / heading situation.